core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -289,7 +289,9 @@ class LogCleaner(val config: CleanerConfig,
           }
       }
       if (!cleaned)
-        backOffWaitLatch.await(config.backOffMs, TimeUnit.MILLISECONDS)
+        if (!backOffWaitLatch.await(config.backOffMs, TimeUnit.MILLISECONDS)) {
+          trace(s"backOffWaitLatch didn't count down within ${config.backOffMs} milliseconds")
```

Comment:
I don't think it makes sense to log this case. This is the expected behaviour. The latch will only count down when `shutdown` is called. To be honest, the existing code is fine, `findBugs` is being a bit too eager in this case. If we want to remove the exclusion, we should probably log the case when `true` is returned.

Reply:
yes, the intention was to remove findbugs exclusion, so that it can uncover bugs in future code changes. Agree, it doesn't make sense to log when false is retuned. Updated the code to log the case when true is returned.